website/docs/r/virtual_machine.html.markdown

```diff
@@ -87,6 +91,7 @@ The following arguments are supported:
 * `nic` - (Optional) Can be specified multiple times to attach several NICs. See [Nic parameters](#nic-parameters) below for details.
 * `vmgroup` - (Optional) See [VM group parameters](#vm-group-parameters) below for details. Changing this argument triggers a new resource.
 * `group` - (Optional) Name of the group which owns the virtual machine. Defaults to the caller primary group.
+* `sched_requirements` - (Optional) A boolean expression to select the hosts that could run the VM.
```

Comment:
I do prefer something like: Scheduling requirements to deploy the resource following specific rule.